apps/alpha-test-app/src/views/HomeView.vue

```diff
@@ -118,12 +119,16 @@ const groupedListboxOptions: ListboxOption[] = [
 
 const multiSelectState = ref(selectOptions.slice(0, 5));
 const singleSelectState = ref(selectOptions[0]);
+
+const toggleDark = useToggle(useDark());
 </script>
 
 <template>
   <OnyxPageLayout>
     <template #sidebar>
       <div class="sidebar">
+        <OnyxButton label="Toggle Dark Mode" @click="toggleDark" />
```

Comment:
I'd prefer to put a switch in the global nav bar inside App.vue so the theme can be toggled on every page

Reply:
do we have the user menu component yet?

Reply:
Yes: https://storybook.onyx.schwarz/?path=/story/components-navbar--with-context-area